android/app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,5 @@
 <resources>
     <string name="app_name">reactNativeApp</string>
+    <string name="facebook_app_id">894605627319646</string>
+    <string name="google_maps_key">AIzaSyALmVEIFj_GhYcKEtpJwtcw40PMEsNqxx0</string>
```

Comment:
Why is this in the **public** versioned code?! Also, why is the Facebook ID needed here?